File: 1000~9999/2490.py
# 윷놀이

import sys

for _ in range(3):
    a = list(map(int, sys.stdin.readline().split()))

    if sum(a) == 4:
        print('E')
    elif sum(a) == 3:
        print('A')
    elif sum(a) == 2:
        print('B')
    elif sum(a) == 1:
        print('C')
    elif sum(a) == 0:
        print('D')

File: 1000~9999/2490.c
// 윷놀이

#include <stdio.h>

int main() {
	int a, b, c, d;
	char n = 3;
	while (n--) {
		scanf("%d %d %d %d", &a, &b, &c, &d);
		int sum = a + b + c + d;
		printf("%c\n", 68 - (sum / 4 * -5 + sum));
	}

	return 0;
}